### ISSUE TITLE
The "can't redeploy non-snapshots" message may not be making it to deploying clients

### ISSUE THREAD

**Issue by tobias** [clojars/clojars-web#774]

**Comment by tobias** [comment]
It looks like something changed between `lein` 2.9.3:

```
Could not transfer metadata org.clojars.tcrawley:deploytest/maven-metadata.xml from/to clojars (https://repo.clojars.org/): Access denied to: https://repo.clojars.org/org/clojars/tcrawley/deploytest/maven-metadata.xml, ReasonPhrase: Forbidden - redeploying non-snapshots is not allowed (see https://git.io/v1IAs).
Failed to deploy metadata: Could not transfer metadata org.clojars.tcrawley:deploytest/maven-metadata.xml from/to clojars (https://repo.clojars.org/): Access denied to: https://repo.clojars.org/org/clojars/tcrawley/deploytest/maven-metadata.xml, ReasonPhrase: Forbidden - redeploying non-snapshots is not allowed (see https://git.io/v1IAs).
```

and 2.9.5:

```
Failed to deploy metadata: Could not transfer metadata org.clojars.tcrawley:deploytest/maven-metadata.xml from/to clojars (https://repo.clojars.org/): Access denied to: https://repo.clojars.org/org/clojars/tcrawley/deploytest/maven-metadata.xml
```

I suspect this may be caused by an upgrade to a version to aether that no longer showing the reason phrase, or has the option to do so but it is now disabled by default. I'll do a bit more digging.

**Comment by tobias** [comment]
This is caused by `lein` `2.9.4` (via `pomegranate`) bringing in `wagon-http` `3.3.2`, where `2.9.3` used `3.0.0`. Displaying the reason phrase was [removed in `3.2.0`](https://issues.apache.org/jira/browse/WAGON-529), then [restored in `3.3.4`](https://issues.apache.org/jira/browse/WAGON-541) when it was realized that it was used to display additional context to the user. 

I'll work on getting `pomegranate` updated to a newer wagon version, then get `lein` updated to that newer `pomegranate`. 

There is also [an issue open to discuss an alternate method of sending this context](https://issues.apache.org/jira/browse/MNG-6795)

**Comment by tobias** [comment]
This was fixed in lein 2.9.7.
